index.html

```diff
@@ -7,7 +7,7 @@
 <body>
 <script>
 var word = "bottles";
-var count = 99;
+var count = 100;
```

Comment:
I believe it would be better if the light was off for 4 seconds and on for just 3